react-accordion-component/src/index.jsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Accordion from './accordion.jsx';
+
+const topics = [
+  {
+    id: 'html',
+    title: 'Hypertext Markup Language',
+    content: 'Hypertext Markup Language (HTML) is the standard markup language' +
+      ' for creating web pages and web application. With Cascading Style Sheets' +
+      ' (CSS) and JavaScript, it forms a triad of cornerstone technologies for the' +
+      ' World Wide Web.'
+  },
```

Comment:
If you want to put a string onto multiple lines you can use back ticks ` `` ` instead of using concatenation, little bit cleaner.

```JSX
{
    id: 'html',
    title: 'Hypertext Markup Language',
    content: `Hypertext Markup Language (HTML) is the standard markup language
      for creating web pages and web application. With Cascading Style Sheets
      (CSS) and JavaScript, it forms a triad of cornerstone technologies for the
      World Wide Web.`
}
```